### PR TITLE
Clean up Derivative API

### PR DESCRIPTION
**Pull request by steven-johnson**

### CODE REVIEW

File: src/Derivative.h [hunk 1]
```diff
@@ -27,21 +27,13 @@ class Derivative {
...
-
-    Func operator()(const Buffer<> &buffer) const {
-        return get(buffer);
-    }
-
-    Func operator()(const Param<> &param) const {
-        return get(param);
-    }
+    /** Get the unbounded Func, if any. If no such Func (ie no boundary condition
+     * was applied), equivalent to just calling get(). */
```
Comment (BachiLi) [inline src/Derivative.h]: equivalent to just calling operator()?

Comment (steven-johnson) [reply src/Derivative.h]: fixed

**Comment by steven-johnson** [comment]
Monday morning review ping

**Comment by steven-johnson** [comment]
Failure is unrelated.

**Pull request merged**